curation_portal/views/app_settings.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class ApplicationSettingsView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        return Response({"settings": {"sign_out_url": settings.CURATION_PORTAL_SIGN_OUT_URL}})
```

Comment:
Can we enforce somewhere that this should be a relative path? Does not seem right to have the option to send back an absolute url.

Reply:
Idk, it seems possible that a sign out URL might point to an IDP vs the service itself.